core/websock.js

```diff
@@ -267,7 +258,7 @@ export default class Websock {
             }
         }
 
-        this._rQlen = this._rQlen - this._rQi;
+        this._rQlen = this.rQlen;
```

Comment:
This is very difficult to make sense of, and has a high risk of breaking in the future. Keep the existing code.

Reply:
Sure thing!